scripts/install_third_party_libs.py

```diff
@@ -0,0 +1,272 @@
+        os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')])
+
+# Download and install future.
+if not os.path.exists(os.path.join('third_party', 'future-0.17.1')):
+    subprocess.call([
+        'pip', 'install', 'future==0.17.1', '--target',
+        os.path.join('third_party', 'future-0.17.1')])
+
+# Download and install psutil.
+if not os.path.exists(os.path.join(TOOLS_DIR, 'psutil-5.6.3')):
+    subprocess.call([
+        'pip', 'install', 'psutil==5.6.3', '--target',
+        os.path.join(TOOLS_DIR, 'psutil-5.6.3')])
+
+# pylint: disable=wrong-import-position
+# pylint: disable=wrong-import-order
+import python_utils  # isort:skip
+
+from . import build  # isort:skip
+from . import common  # isort:skip
+from . import install_third_party  # isort:skip
+from . import pre_commit_hook  # isort:skip
+from . import pre_push_hook  # isort:skip
+from . import setup  # isort:skip
+from . import setup_gae  # isort:skip
+# pylint: enable=wrong-import-order
+# pylint: enable=wrong-import-position
+
+_PARSER = argparse.ArgumentParser(description="""
+    Installation script for Oppia third-party libraries.""")
+
+_PARSER.add_argument(
+    '--nojsrepl',
+    help='optional; if specified, skips installation of skulpt.',
+    action='store_true')
+_PARSER.add_argument(
+    '--noskulpt',
+    help='optional; if specified, skips installation of skulpt.',
+    action='store_true')
+
+
+def pip_install(package, version, install_path):
+    """Installs third party libraries with pip.
+
+    Args:
+        package: str. The package name.
+        version: str. The package version.
+        install_path: str. The installation path for the package.
+    """
+    try:
+        python_utils.PRINT('Checking if pip is installed on the local machine')
+        # Importing pip just to check if its installed.
+        import pip  #pylint: disable=unused-variable
+    except ImportError:
+        common.print_each_string_after_two_new_lines([
+            'Pip is required to install Oppia dependencies, but pip wasn\'t '
+            'found on your local machine.',
+            'Please see \'Installing Oppia\' on the Oppia developers\' wiki '
+            'page:'])
+
+        os_info = os.uname()
+        if os_info[0] == 'Darwin':
+            python_utils.PRINT(
+                'https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Mac-'
+                'OS%29')
+        elif os_info[0] == 'Linux':
+            python_utils.PRINT(
+                'https://github.com/oppia/oppia/wiki/Installing-Oppia-%28Linux'
+                '%29')
+        else:
+            python_utils.PRINT(
+                'https://github.com/oppia/oppia/wiki/Installing-Oppia-%28'
+                'Windows%29')
+        raise Exception
+
+    subprocess.call([
+        'pip', 'install', '%s==%s' % (package, version), '--target',
+        install_path])
+
+
+def install_skulpt(args):
+    """Download and install Skulpt. Skulpt is built using a Python script
```

Comment:
Move this into _PARSER's description

Reply:
This is a function docstring not the overview of what the script does, so shouldn't is just stay here?

Reply:
True, ok fine with me.